Source/WebCore/platform/graphics/gbm/DMABufReleaseFlag.h

```diff
@@ -72,7 +72,8 @@ struct DMABufReleaseFlag {
             return;
 
         uint64_t value { 1 };
-        write(fd.value(), &value, sizeof(uint64_t));
+        if (write(fd.value(), &value, sizeof(value)) != sizeof(value))
+            WTFLogAlways("Error writing to the eventfd at DMABufReleaseFlag");
```

Comment:
You can `#include <wtf/SafeStrerror.h>`, then:

```suggestion
            WTFLogAlways("Error writing to the eventfd at DMABufReleaseFlag: %s", safeStrError(errno));
```

Reply:
Nice stuff! will include it :)